docs/structured-streaming-kafka-integration.md

```diff
@@ -376,6 +383,8 @@ The following configurations are optional:
 </tr>
 </table>
 
+If you would like to disable the caching for Kafka consumers, you can set `spark.streaming.kafka.consumer.cache.enabled` to `false`. Disabling the cache may be needed to workaround the problem described in SPARK-19185. This property may be removed in later versions of Spark, once SPARK-19185 is resolved.
```

Comment:
I don't think we really need to indicate we're going to remove it later - the burden of keeping it around is low, and I don't expect we'd ever get around to removing it anyway.

Reply:
This is the description in dstreams and I've made it consistent by copying it. Should we remove it then from dstreams as well? Maybe I'm too optimistic but considering this as a work-around as it's stated in the descriptions.